core/iwasm/compilation/aot_llvm.h

```diff
@@ -114,8 +114,6 @@ typedef struct AOTValueSlot {
 typedef struct AOTCompFrame {
     /* The current wasm module */
     WASMModule *cur_wasm_module;
-    /* The current wasm function */
-    WASMFunction *cur_wasm_func;
     /* The current wasm function index */
     uint32 cur_wasm_func_idx;
```

Comment:
Can also remove cur_wasm_func_idx, it isn't used also.

Reply:
Removed, thanks